components/coding/studentPortal/ProfileDetailCard.tsx

```diff
@@ -21,34 +22,55 @@ function ProfileDetailCard({
   badges,
   totalBadgeCount,
   currentBadge,
-  nextGoal,
+  completedGoal,
+  completedDate,
 }: ProfileDetailCard) {
   return (
-    <div className="flex flex-col h-full gap-4 text-sm border-2 bg-slate-50 text-slate-800 border-slate-800 hover:bg-violet-100">
-      <div className="flex flex-col items-center">
+    <div
+      className="grid grid-cols-4 h-full w-full text-sm border-2 bg-slate-50 text-slate-800 border-slate-800 hover:bg-violet-100"
+      style={{ height: "340px", objectFit: "contain" }}
```

Comment:
I took this line out and everything looked the same -- are you sure we need this?

trying to stay away from absolute sizing as much as possible since that can cause issues for responsiveness on different screen sizes

Reply:
I took it out -- only if we want to ensure if the text goes longer on the goal that the containers will always remain the same height.  I do agree with you, though, to take it out because the circle is big enough it makes the spacing solidly larger than we need so I removed it!